app/jobs/onceoff/rebake_gif_images.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Jobs
+  class RebakeGifImages < ::Jobs::Onceoff
+    def execute_onceoff(args)
+      Upload.where("original_filename LIKE '%.gif'").find_each do |upload|
+        uri = Discourse.store.path_for(upload) || upload.url
+        upload.update!(animated: FastImage.animated?(uri))
```

Comment:
Should we catch any exceptions here?

Reply:
I do not think so. What exceptions do you have in mind?

FastImage should not throw (there is an option for that, disabled by default) and if AR throws then there is something more serious going on and we should see that in logs. If a onceoff job throws, it seems that it is repeated until it succeeds, which  is desired behavior.